content/en/docs/tasks/security/authentication/authn-policy/index.md

```diff
@@ -388,16 +414,14 @@ spec:
 EOF
 {{< /text >}}
 
-Get ingress IP
-
-{{< text bash >}}
-$ export INGRESS_HOST=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
-{{< /text >}}
+Follow the instructions in
+    [Determining the ingress IP and ports](/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports)
+to define the `INGRESS_HOST` and `INGRESS_PORT` environment variables.
 
 And run a test query
 
 {{< text bash >}}
-$ curl $INGRESS_HOST/headers -s -o /dev/null -w "%{http_code}\n"
+$ curl "$INGRESS_HOST":"$INGRESS_PORT"/headers -s -o /dev/null -w "%{http_code}\n"
```

Comment:
I think it's cleaner to just wrap the entire URL argument in quotes:
```suggestion
$ curl "$INGRESS_HOST:$INGRESS_PORT/headers" -s -o /dev/null -w "%{http_code}\n"
```

Reply:
Agreed.  I pushed a version with that change.